crates/ruma-common/src/api.rs

```diff
@@ -232,6 +232,9 @@ pub use ruma_macros::ruma_api;
 ///   not just JSON. The field type must be `Vec<u8>`.
 pub use ruma_macros::request;
 
+// TODO: Write docs
```

Comment:
I'd like to see those docs before this gets merged.

Reply:
Oops 😇